src/odemis/util/test/linalg_test.py

```diff
@@ -71,5 +72,54 @@ def testUnity(self):
             self.assertTrue(numpy.allclose(tri_inv(c), c))
 
 
+class PlaneFittingTestCase(unittest.TestCase):
+    def test_fit_plane_lstsq(self):
+        """
+        Test fitting a plane to a set of 3D coordinates using least-squares fitting.
+        """
+        # Create a plane with equation z = 2x + 3y + 4
```

Comment:
I think you can combine this comment with the one on line 82, because this equation is not valid anymore for the second part of this test case, or is it?

Reply:
The points calculated in the first and second part both lie on this plane. First part tests when points are linear, many plane equations can fit through them. Check that c is not infinity. Second part tests the plane equation through non-planar points,  which would give a unique plane equation.